plugins/accessibility/index.js

```diff
@@ -0,0 +1,88 @@
+var q = require('q'),
+    fs = require('fs'),
+    path = require('path'),
+    _ = require('lodash');
+
+var AUDIT_FILE = path.join(__dirname, '../../node_modules/accessibility-developer-tools/dist/js/axs_testing.js');
+
+/**
+ * You enable this plugin in your config file:
+ *
+ *    // Chrome Accessibility Developer Tools:
+ *    exports.config = {
+ *      ...
+ *      plugins: {
+ *        ...
+ *        accessibility: {
+ *          developerTools: true,
```

Comment:
Can  you add a TODO comment mentioning that developerTools is the only thing that will be run at the moment, since the config isn't actually used yet?